### PR TITLE
[IA-2545] Update Galaxy default conf to 1 n1-highmem-8 node

### PR DESCRIPTION
**Pull request by rtitle**
See rationale in JIRA ticket: https://broadworkbench.atlassian.net/browse/IA-2545

See corresponding Leo PR: https://github.com/DataBiosphere/leonardo/pull/1893

I tested locally. With the change:
image

Current dev:
image

Nice cost savings for the user too.

### CODE REVIEW

File: src/components/NewGalaxyModal.js [hunk 1]
```diff
@@ -203,8 +203,8 @@ export const NewGalaxyModal = _.flow(
   }
 
   const renderDefaultCase = () => {
-    const { cpu, memory } = _.find({ name: 'n1-standard-8' }, machineTypes)
```
Comment (Qi77Qi) [inline src/components/NewGalaxyModal.js:206]: once we allow users to specify machine type, this needs to be updated as well right...not necessarily in this PR, but might deserve a ticket

**Review by Qi77Qi — APPROVED** [review]

File: src/components/NewGalaxyModal.js (hunk 1 above)
Comment (rtitle) [reply src/components/NewGalaxyModal.js:206]: Yeah - this currently just displays the default, but the UI will eventually need to allow users to choose their machine type and number of nodes.

Comment (Qi77Qi) [reply src/components/NewGalaxyModal.js:206]: I see..this is just the defaults

**Comment by rtitle** [comment]
Since Terra UI deploys daily, this should be coordinated with the Leo deploy of https://github.com/DataBiosphere/leonardo/pull/1893

File: src/components/NewGalaxyModal.js [hunk 2]
```diff
@@ -203,8 +203,8 @@ export const NewGalaxyModal = _.flow(
...
-    const cost = getGalaxyCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8', numNodes: 2 } })
+    const { cpu, memory } = _.find({ name: 'n1-highmem-8' }, machineTypes)
+    const cost = getGalaxyCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-highmem-8', numNodes: 1 } })
```
Comment (kyuksel) [inline src/components/NewGalaxyModal.js]: Might be good to factor out the default machine type so there's one place to change going forward:

```javascript
const defaultMachineType = 'n1-highmem-8'
```

**Comment by kyuksel** [comment]
@rtitle While reviewing your PR I noticed that we've also not been factoring `numNodes` into the `Cloud Compute size` display:

```javascript
span({ style: { fontWeight: 600 } }, [`${cpu} CPUS, ${memory} GB of memory, 250 GB disk space`])
```

The information displayed on the page will be correct after your PR is merged since the new default config will include a single node instead of two but it would be good to make that dynamic in case the default config has two nodes again in the future. Not sure if this should loop in the UX team as well.

What do you think?

**Comment by rtitle** [comment]
@kyuksel yeah me and @almahmoud noticed that too: the UI doesn't actually display the number of nodes in the `NewGalaxyModal`. However the number of nodes _is_ taken into account in the cost calculation (https://github.com/DataBiosphere/terra-ui/blob/e0e46a981e6b9355456ab036a73197b84cec5202/src/libs/runtime-utils.js#L109).

Since this code be changing anyway (I imagine) when we add compute configurability, I'm inclined to leave this PR as is to just make the default case correct for now.

**Comment by kyuksel** [comment]
> Since this code be changing anyway (I imagine) when we add compute configurability

Yea I can look into this as part of the compute configurability ticket.

**Review by kyuksel — APPROVED** [review]

**Pull request merged**